docs/pages/choose-an-edition/teleport-enterprise/gcp-kms.mdx

```diff
@@ -21,13 +21,10 @@ In short, private key material will never leave Google Cloud KMS.
 If launching a new Teleport cluster this will all be handled during initial
 startup with no specific interventions required after configuration.
 For existing Teleport clusters that already have private key material on disk,
-a CA rotation must be performed. This ensures that the private key material
-only ever exists in KMS when this feature is enabled.
+a CA rotation must be performed.
 Read on to [migrating an existing cluster](#migrating-an-existing-cluster) to
 learn more.
 
-(!docs/pages/includes/cloud/call-to-action.mdx!)
-
```

Comment:
This CTA reads "Teleport Team takes care of this setup for you so you can provide secure access to your infrastructure right away". I find that a bit disingenuous when Teleport Team doesn't use this feature at all, and getting the same CA key configuration as Team takes 0 extra steps